.github/config/evergreen-arm-hardfp.json

```diff
@@ -1,10 +1,19 @@
 {
   "docker_service": "build-raspi",
   "bootloader": "raspi-2",
-  "on_device_test": "true",
+  "on_device_test": {
+    "enabled": true,
+    "tests": [
+      "evergreen_test"
```

Comment:
What about unit and black box tests for evergreen-arm-hardfp platform?

Reply:
I thought evergreen_test is the only flavor here, I'll add them now.

Reply:
resolved